docs/commands/_category_.json

```diff
@@ -1,4 +0,0 @@
-{
```

Comment:
[Question] Why - better to autogenerate that?

Reply:
Sorry I don't fully understand your question.

Reply:
Why file is removed vs we have that info in website. More of question so I will understand it.

Reply:
https://github.com/craicoverflow/app-services-cli/blob/ux-guidelines-base-structure/website/sidebars.js#L24-L30

I have added this to the sidebars file instead which does the same thing. I wanted to have the contributing section open by default and this one collapsed, and wanted more control over ordering.

Reply:
Really cool! Thank you for explaining